app/models/manageiq/providers/amazon/cloud_manager/refresh_parser.rb

```diff
@@ -192,8 +190,8 @@ def parse_image(image, is_public)
       :vendor             => "amazon",
       :raw_power_state    => "never",
       :template           => true,
-      :labels             => labels,
-      :tags               => map_labels('Image', labels),
+      :labels             => parse_labels(image.tags),
+      :tags               => map_labels('Image', parse_tags(image.tags)),
```

Comment:
Hmm, not sure, AFAICT it's equivalent to previous version which IMO was clearer.
`:labels => parse_labels(...)` will eventually be passed to save_labels_inventory, and will result in saving into "labels" section.  `parse_tags` here feeds same info with different :section into `map_labels()`, but `map_labels` doesn't look at section at all.

What I meant is that maybe it makes more sense to actually save AWS tags into custom_attributes with a different section e.g. "aws_tags" or "provider_tags" or whatever.  *IF* you want that, you can do it (map_labels doesn't care).